twarep/electrifly
# new menu styling with dropdowns and fixed naming convention

## Checklist
**Note:** Put an x into the box like this [x] to check these off

## What I Did
Restyled app.py to have menu bars and readded the original bootstrap. Changed all deprecated components

## How I Did it?
Looked at Shiny documentation

## How to Test?
Run app.py and make sure all functionalities work as expected. The 2-tab system should not be visible as it's all within menu dropdowns now.
<img width="1488" alt="image" src="https://github.com/twarep/electrifly/assets/64234627/8c7bf5ef-2571-4e25-918c-db846e172aef">


### Relevant Links

- [Card with a pill tabset](https://shiny.posit.co/py/layouts/tabs/#tabset-with-pill-list-navigation)
